test/test_helper.rb

```diff
@@ -2,3 +2,79 @@
 require 'tyrant'
 
 require 'minitest/autorun'
+
+require 'warden'
+require 'rack-minitest/test'
+
+# Use Warden spec helper to simulate Warden
+# https://github.com/hassox/warden/blob/master/spec/helpers/request_helper.rb
+module Warden::Spec
```

Comment:
Don't you think we could use this module directly from Warden, without copying it?